pkg/users/account-create-dialog.js

```diff
@@ -154,6 +170,10 @@ function validate_uid(uid, accounts, min_uid, max_uid, change) {
     }
 }
 
+function validate_home_dir(dir) {
+    return cockpit.spawn(["test", "!", "-d", dir], { superuser: "require" });
```

Comment:
This `test` will also fail if `dir` exists, but is not actually a dir. That would look a bit confusing? That's why I wanted these cases in the integration test as well.

Reply:
As discussed in IRC, let's solve this in follow-up PR